### DISCUSSION TITLE
Copilot proposes syntactically incorrect code

### DISCUSSION THREAD

**Discussion by stefaanMLB** [community/community#50091] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Copilot routinely proposes syntactically incorrect code.
This is quite annoying.  I prefer not to get a proposal rather than getting incorrect ones.
Would be nice if copilot could communicate with the language server to avoid giving syntactically incorrect proposals.
Here's a screenshot. Copilot just proposed the red underlined code.
![image](https://user-images.githubusercontent.com/50827063/225254732-1f9a8a90-c356-48a6-82b3-34001af55de6.png)

**Comment by stefaanv** [comment]
Is no-one else bothered by this ?
Or is copilot not being used by anyone ?
This drives me mad !
